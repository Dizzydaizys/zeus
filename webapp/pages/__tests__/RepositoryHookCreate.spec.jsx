import React from 'react';
import {render} from 'enzyme';
import sinon from 'sinon';

import {RepositoryHookCreate} from '../RepositoryHookCreate';

describe('RepositoryHookCreate', () => {
  it('renders', () => {
    const tree = render(
      <RepositoryHookCreate addIndicator={sinon.spy()} removeIndicator={sinon.spy()} />,
      {
        context: {repo: TestStubs.Repository(), router: TestStubs.router()}
      }
    );
    expect(tree).toMatchSnapshot();
  });
});
