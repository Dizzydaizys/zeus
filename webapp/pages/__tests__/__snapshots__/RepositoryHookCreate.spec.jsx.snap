// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepositoryHookCreate renders 1`] = `
<div>
  <div>
    <div>
      <div
        class="sc-bxivhb Nbskm"
      >
        Create Hook
      </div>
    </div>
    <p>
      Hooks lets you easily upsert build information into Zeus. They're a set of credentials that are bound to this repository. Once you've created the hook, it can be used to submit build and job information, as well as to upload artifacts (for example, via 
      <a
        href="https://github.com/getsentry/zeus-cli"
      >
        zeus-cli
      </a>
      ).
    </p>
    <p>
      To create a new hook, select a provider:
    </p>
    <p>
      <a
        class="sc-htpNat jfsIGo"
      >
        Travis CI
      </a>
       
      <a
        class="sc-htpNat jfsIGo"
      >
        Custom
      </a>
    </p>
  </div>
</div>
`;
